ktorm-support-mysql/src/test/kotlin/org/ktorm/support/mysql/MySqlTest.kt

```diff
@@ -26,6 +26,16 @@ import java.util.concurrent.TimeoutException
 class MySqlTest : BaseTest() {
 
     companion object {
+        const val TOTAL_RECORDS = 4
+        const val MINUS_ONE = -1
+        const val ZERO = 0
+        const val ONE = 1
+        const val TWO = 2
+        const val ID_1 = 1
+        const val ID_2 = 2
+        const val ID_3 = 3
+        const val ID_4 = 4
```

Comment:
We don't have to define these numbers as constants. It makes no sense. Seems the quality check doesn't ignore the test code?

Reply:
Yep, it doesn't